Kitodo/src/test/java/org/kitodo/services/data/RulesetServiceIT.java

```diff
@@ -127,10 +129,10 @@ public void shouldFindByTitle() throws Exception {
     public void shouldFindByFile() throws Exception {
         RulesetService rulesetService = new RulesetService();
 
-        JSONObject ruleset = rulesetService.findByFile("ruleset_slubdd.xml");
+        JSONObject ruleset = rulesetService.findByFile("ruleset_slubhh.xml");
```

Comment:
why rename it from slubdd to slubhh?
ruleset_test, like in the other place, is way better

Reply:
because slubdd was renamed to test, I didn't want to use real slubdd file in test

Reply:
so rename it to something without slub OR hh in it.
ruleset_title_test... whatever...

Reply:
done